src/theme/components/ToggleButtonTheme.ts

```diff
@@ -0,0 +1,48 @@
+import { Components, Theme } from "@mui/material"
+
+export default function ToggleButtonTheme(theme: Theme): Components {
+  console.log("theme", theme)
+  return {
+    MuiToggleButton: {
+      styleOverrides: {
+        root: {
+          textTransform: "none",
+          color: theme.palette.getContrastText(theme.palette.mute.main),
+          "&.Mui-selected": {
+            backgroundColor: theme.palette.action.active,
+            border: `1px solid ${theme.palette.primary.main} !important`,
```

Comment:
does this styling not work without `!important`?

Reply:
Very good question.
Actually, it is not good to use "!important". 

As you can see from the following screenshot, the border-left was having transparent on MuiToggleButtonGroup. 
![Screen Shot 2022-02-02 at 2 11 31 PM](https://user-images.githubusercontent.com/75422800/152151335-593b4b74-82d1-40c1-9d53-db45c924365b.png)

To override border-left. I gave Mui-selected on MuiToggleButton. But because border-left was defined MuiToggleButtonGroup, so after giving border on the MuiToggleButton, border-left didn't overrided.

Now, I made "Mui-selected" class on both MuiToggleButton and MuiToggleButtonGrop, so overrides the border-left.
Current Css is the following.

![Screen Shot 2022-02-02 at 2 24 38 PM](https://user-images.githubusercontent.com/75422800/152153279-6971ddcb-b813-4e8c-ac77-76342841b8ae.png)